compass-js-frontend/src/main/webapp/app.js

```diff
@@ -23,6 +24,7 @@ app.extend({
 		app.router = new CompassRouter();
 		app.basePath = "/";
 		app.projects = new ProjectCollection();
+		app.scenenodes = new SceneNodeCollection();
```

Comment:
Should the scenenodes really be a member of the app? I would expect to have a single root node within a scenario.